lib/dom.mli

```diff
@@ -103,426 +103,1403 @@ val forwardRef : ('props -> domRef -> Core.element) -> 'props Core.component
     let forwardRef renderFunc = forwardRef_internal Imports.react renderFunc]
 
 module Style : sig
+  [@@@js.stop]
+
+  type style_key
+
   type t = style
```

Comment:
If we want to use proper CSS terminology, `style_key` would be a "declaration" ("decl" for short), and `style` would be a "declaration block".